index.js

```diff
@@ -759,3 +759,28 @@ is.hex = function (value) {
 is.symbol = function (value) {
   return typeof Symbol === 'function' && toStr.call(value) === '[object Symbol]' && typeof symbolValueOf.call(value) === 'symbol';
 };
+
+is.not = negateMethods(is);
+
+function negateMethods(source) {
+	var negatedMethods = {};
+	Object.keys(source).forEach(function (key) {
```

Comment:
another alternative, to avoid adding dependencies, might be to just `is.not.symbol = function (value) { return !is.symbol(value); }` etc, manually, next to each method.